scio-core/src/main/scala/com/spotify/scio/ScioResult.scala

```diff
@@ -176,4 +178,8 @@ class ScioResult private[scio] (val internal: PipelineResult,
     aggregators.getOrElse(acc.name, Nil)
```

Comment:
So this is the method that actually throws the exception IIRC. How about `require` here? This is also more future proof in case there is another methods that wants to access `internal.getAggregatorValues`? What do you think?